tests/PasswordTest.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace atk4\data\tests;
+
+use atk4\data\Model;
+use atk4\data\Persistence_Array;
+
+class PasswordTest extends TestCase
+{
+    public function testPasswordField()
+    {
+        $m = new Model(); //$db, 'job');
+
+        $m->addField('p', ['Password']);
+
+        $m['p'] = 'mypass';
+
+        // when setting password, you cannot retrieve it back
+        $this->assertEquals('mypass', $m['p']);
```

Comment:
so this test should fail, because `m['p']` should be `null`.

Reply:
only after it's saved. Otherwise we can't save it.